extensions/mentions/tests/integration/api/UserMentionsTest.php

```diff
@@ -488,9 +489,11 @@ public function editing_a_post_with_deleted_author_that_has_a_mention_works()
             ])
         );
 
-        $this->assertEquals(200, $response->getStatusCode());
+        $body = $response->getBody()->getContents();
```

Comment:
Maybe we should make a helper function for this/

Reply:
I actually plan to make a separate PR that improves the testing experience, rather than a helper function I'm thinking of creating a `TestReponse` that is essentially a wrapper around the response, and allows similar laravel useful methods like

```php
$response->assertOk()
$response->assertStatus(201);
$response->body()
...
```

It would make a huge difference from my experience writing tests with Laravel VS Flarum.